src/main/kotlin/org/opensearch/replication/action/stop/TransportStopIndexReplicationAction.kt

```diff
@@ -93,7 +94,13 @@ class TransportStopIndexReplicationAction @Inject constructor(transportService:
                     throw OpenSearchException("Failed to remove index block on ${request.indexName}")
                 }
 
-                validateReplicationStateOfIndex(request)
+                try {
+                    validateReplicationStateOfIndex(request)
```

Comment:
Let's make sure, this doesn't affect the rest of the operations.

Reply:
Verified, this does not affect any other operations.